mteb/models/colbert_models.py

```diff
@@ -225,3 +225,35 @@ def similarity(self, a: np.ndarray, b: np.ndarray) -> np.ndarray:
         "MIRACL": ["train"],
     },
 )
+
+
+lightonai__GTE_ModernColBERT_v1 = ModelMeta(
+    loader=partial(  # type: ignore[call-arg]
+        ColBERTWrapper,
+        model_name="lightonai/GTE-ModernColBERT-v1",
+    ),
+    name="lightonai/GTE-ModernColBERT-v1",
+    languages=[
+        "eng-Latn",  # English
+    ],
+    open_weights=True,
+    revision="78d50a162b04dfdc45c3af6b4294ba77c24888a3",
+    public_training_code=None,
```

Comment:
the code has been shared [here](https://gist.github.com/NohTow/3030fe16933d8276dd5b3e9877d89f0f)
I can add it directly to the PyLate repository if that's better!

Reply:
This is fine. If you want to improve reproducibility, then I would add the [script dependencies](https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies) and the revisions for the datasets.

```suggestion
    public_training_code="https://gist.github.com/NohTow/3030fe16933d8276dd5b3e9877d89f0f",
```